data-plane/config/broker/500-receiver.yaml

```
@@ -212,6 +211,10 @@ spec:
       port: 8080
       protocol: TCP
       targetPort: 8080
+    - name: https
```

https already exists, I think you meant to have this?
```suggestion
    - name: https-container
```